src/main/java/uk/gov/companieshouse/api/accounts/utility/filetransfer/AmazonS3HandlerImpl.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.api.accounts.utility.filetransfer;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.Protocol;
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AmazonS3HandlerImpl implements AmazonS3Handler {
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public AmazonS3 getAmazonS3(String awsAccessKeyId, String awsSecretAccessKey,
+        String cloudProxyHost, Integer cloudProxyPort) {
+
+        AWSCredentials credentials = new BasicAWSCredentials(awsAccessKeyId, awsSecretAccessKey);
```

Comment:
The client should attempt to find credentials by default, it will implicitly looks for the aws environment variables so BasicAWSCredentials need not be supplied.
The AmazonS3Client is deprecated in later versions of the sdk (our version is over 2 years old). its probably better to use the credentials builder
```
return AmazonS3ClientBuilder
            .standard()
            .withClientConfiguration(getClientConfiguration(cloudProxyHost, cloudProxyPort))
            .build();
```

Reply:
I will try that